ci/azure/posix.yml

```diff
@@ -56,15 +56,20 @@ jobs:
   steps:
     - script: |
         if [ "$(uname)" == "Linux" ]; then sudo apt-get install -y libc6-dev-i386 $EXTRA_APT; fi
-        echo '##vso[task.prependpath]$(HOME)/miniconda3/bin'
+        echo '##vso[task.setvariable variable=MINICONDA_DIR]$(HOME)/miniconda3'
```

Comment:
I think this needs to be executed by `bash` and not `script` (which on windows is just cmd.exe). Not sure if there is an overarching reason for this and similar blocks to be `script` but as mentioned changing to `bash` might fix the issue remaining